forms/serializers/form.py

```diff
@@ -170,8 +171,7 @@ def update(self, instance, validated_data):
         return instance
 
 
-class FormSerializer(serializers.ModelSerializer):
-
+class CreateFormSerializer(serializers.ModelSerializer):
```

Comment:
FormViewSet does not have "create" action. (mixins.CreateModelMixin) Therefore this is named maybe a littlebit funky?

Reply:
Maybe UpdateFormSerializer then